src/samples/flow/netctl/list-network-services.fbp

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env sol-fbp-runner
+
+# This file is part of the Soletta™ Project
+#
+# Copyright (C) 2015 Intel Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This program will print to console a list of all network services
+#
+# Examples:
+#  * list all network services
+#    $ ./list_network_services.fbp
+
+list(netctl/list-services) OUT -> IN _(console:prefix="Service: ")
```

Comment:
since I've requested 2 ports, you do the following:

```sh
list(netctl/list-services)
list ADDED -> IN _(console:prefix="Added: ")
list REMOVED -> IN _(console:prefix="Removed: ")
```